ibm/service/codeengine/resource_ibm_code_engine_app.go

```diff
@@ -752,6 +751,10 @@ func resourceIbmCodeEngineAppUpdate(context context.Context, d *schema.ResourceD
 	hasChange := false
 
 	patchVals := &codeenginev2.AppPatch{}
+	if d.HasChange("project_id") {
```

Comment:
We already defined ForceNew: true on "project_id" we don't need to add this condition

Reply:
Removed, thanks for that.